src/clarity/dropdown/_dropdown.clarity.scss

```diff
@@ -35,7 +35,7 @@ $dropdown-white: clr-getColor(lightest);
                 margin: 0;
             }
 
-            clr-icon[shape="caret"] {
+            clr-icon[shape~="caret"] {
```

Comment:
Nitpick: I think `[shape^="caret"]` is a tiny bit better here.
Same for the couple other ones further in that file.